versions/3.0.md

```diff
@@ -1606,29 +1540,38 @@ Rate-limit headers:
 {
     "X-Rate-Limit-Limit": {
         "description": "The number of allowed requests in the current period",
-        "type": "integer"
```

Comment:
This was a tough call that we went back and forward on just a couple of weeks ago.  Eventually, we sided on using the schema object for consistency.  Yes it is more verbose, however making a clear delineation between what are parameter specific properties and what are schema related properties helps in understanding the behavior of a parameter.  

It also has the side benefit of simplifying the documentation and tooling implementation.